core/src/navigation/LeftNav.html

```diff
@@ -164,22 +220,42 @@
   export default {
     data() {
       return {
-        virtualGroupPrefix: NavigationHelpers.virtualGroupPrefix
+        virtualGroupPrefix: NavigationHelpers.virtualGroupPrefix,
+        selectedCategory: null
       };
     },
     oncreate() {
+      const responsiveNavSetting = LuigiConfig.getConfigValue(
+        'settings.responsiveNavigation'
+      );
+      const semiCollapsible = responsiveNavSetting === 'semiCollapsible';
       this.set({
         hideNavComponent: LuigiConfig.getConfigBooleanValue(
           'settings.hideNavigation'
         ),
-        expandedCategories: NavigationHelpers.loadExpandedCategories()
+        expandedCategories: NavigationHelpers.loadExpandedCategories(),
+        responsiveNavSetting,
+        previousWindowWidth: window.innerWidth,
+        semiCollapsible
       });
 
       StateHelpers.doOnStoreChange(this.store, () => {
         this.set({
           footerText: LuigiConfig.getConfigValue('settings.sideNavFooterText')
         });
       });
+
+      // set isSemiCollapsed to true for mobile
```

Comment:
Check with @hardl if the responsiveNav setting should be able to be changed during runtime. If yes, the following code should be moved into the `onStoreChange` block.

Reply:
@hardl said he sees no usecase where you would change that on runtime.